src/test/java/AndreeaG/tests/tema5/Demoqa/Buttons.java

```diff
@@ -0,0 +1,29 @@
+package AndreeaG.tests.tema5.Demoqa;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import teofilursan.driver.BrowserManager;
+
+public class Buttons {
+
+    public static void main(String[] args) {
+     clickButtons();
+    }
+    public static void clickButtons(){
```

Comment:
It's ok to leave one blank line between methods. So please make also this change in all classes. Thanks!

Reply:
Fingers crossed :)